org.free_astro.siril.json

```diff
@@ -94,13 +94,14 @@
             ],
             "config-opts": [
                 "-D CMAKE_BUILD_TYPE=RELEASE",
-                "-D BUILD_EXAMPLES=OFF"
+                "-D BUILD_EXAMPLES=OFF",
+                "-D OPENCV_GENERATE_PKGCONFIG=ON"
```

Comment:
This is the error. It should be `"-DOPENCV_GENERATE_PKGCONFIG=ON"` (note the lack of space after `-D`). Also same for the previous lines.

I'm not sure why CMake doesn't error.

(I tested and the build is successfull)

Reply:
For the record I'm to blame for the original syntax error. Probably copy-pasta from elsewhere.

Reply:
No. I'm not a cmake user, I did not know.
I've fixed it, it looks ok now.